### ISSUE TITLE
Replace defaultProps with default parameters

### ISSUE THREAD

**Issue by allanlasser** [rexxars/react-refractor#31]
Thanks for this library!

I'm shown this warning at runtime with React 18:

```
Warning: Refractor: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.
```

**Comment by Evavic44** [comment]
Until the issue is fixed, you can silently disable the warning from the console.

```js
const error = console.error;
  console.error = (...args: any) => {
    if (/defaultProps/.test(args[0])) return;
    error(...args);
  };
```

**Comment by piscopancer** [comment]
is it fixed? or what should I do to disable it completely?

**Comment by rexxars** [comment]
Fixed in [v3.0.0](https://github.com/rexxars/react-refractor/releases/tag/v3.0.0).
Note that this module is now ESM-only.
